### PR TITLE
Update translations for - Tigrinya, Urdu, Vietnamese and Yoruba

### LINKED ISSUE #3721: Update translations for services
**Is your feature request related to a problem? Please describe.**
We have updated translations for these services:

Azeri
Japanese
Kyrgyz
https://github.com/bbc/simorgh/pull/3817

---

Afaan Oromoo 
Afrique
Amharic 
Arabic
Gahuza
https://github.com/bbc/simorgh/pull/3841

---

Hausa
Igbo
Indonesian
Korean
Pashto
https://github.com/bbc/simorgh/issues/3830

---

Persian
Pidgin 
Punjabi
Somali
Swahili
Thai
https://github.com/bbc/simorgh/pull/3823

---

Tigrinya
Urdu
Vietnamese 
Yoruba 
https://github.com/bbc/simorgh/pull/3821

We should update Simorgh so that it has all of the translations. Note that **Cookie and Privacy banner translations will be in a separate issue: https://github.com/bbc/simorgh/issues/3722**

**Describe the solution you'd like**
- Update the service configs at `src/app/lib/config/services/:service.js`
- Check the Simorgh pages load correctly: localhost urls are here: https://github.com/bbc/simorgh/wiki/Simorgh-Pages 

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Before putting the PR through code review, the full cypress test suite should be run (all page types for all services) with `CYPRESS_SMOKE=false npm run test:e2e`

- [ ] This feature is expected to need manual testing.

**Additional context**
N/A

### LINKED ISSUE #3806: Update translations for - Tigrinya, Urdu, Vietnamese and Yoruba
Partly completes: #3721 

**Is your feature request related to a problem? Please describe.**
We have updated translations for these services:
Tigrinya
Urdu
Vietnamese
Yoruba

We should update Simorgh so that it has all of the translations. **Note that Cookie and Privacy banner translations will be in a separate issue**: #3722

**Describe the solution you'd like**

Update the service configs at src/app/lib/config/services/:service.js
Check the Simorgh pages load correctly: localhost urls are here: https://github.com/bbc/simorgh/wiki/Simorgh-Pages

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Before putting the PR through code review, the full cypress test suite should be run (all page types for all services) with CYPRESS_SMOKE=false npm run test:e2e

- [ ] This feature is expected to need manual testing.

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by DenisHdz**
Resolves https://github.com/bbc/simorgh/issues/3806
Partly resolves #3721 and #3722

**Overall change:** 
Update translations for Tigrinya, Urdu, Vietnamese and Yoruba.

**Code changes:**
- Update `src/app/lib/config/services/<service>.js` for Tigrinya, Urdu, Vietnamese and Yoruba.

---

- [X] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [X] This PR requires manual testing

### CODE REVIEW

**Review by AlistairGempf — APPROVED** [review]
Looks good to me 👍

File: src/app/lib/config/services/tigrinya.js [hunk 1]
```diff
@@ -11,20 +11,20 @@ const service = {
...
     defaultImage: 'https://news.files.bbci.co.uk/ws/img/logos/og/tigrinya.png',
     defaultImageAltText: 'BBC News ትግርኛ',
     dir: `ltr`,
-    externalLinkText: ', external',
-    imageCaptionOffscreenText: 'Image caption, ',
-    videoCaptionOffscreenText: 'Video caption, ',
-    audioCaptionOffscreenText: 'Audio caption',
-    defaultCaptionOffscreenText: 'Caption, ',
-    imageCopyrightOffscreenText: 'Image source, ',
+    externalLinkText: ', ደጋዊ',
```
Comment (AlistairGempf) [inline src/app/lib/config/services/tigrinya.js]: I'm not sure if these should have a comma. In the spreadsheet there isn't one (and the others end with a colon instead)

File: src/app/lib/config/services/tigrinya.js [hunk 2]
```diff
@@ -43,11 +43,11 @@ const service = {
       brandLogoColour: `${C_WHITE}`,
     },
     translations: {
-      seeAll: 'See all',
+      seeAll: 'ኩሉ ረኣይ',
       home: 'ዜና',
-      currentPage: 'Current page',
+      currentPage: 'ህሉው ገጽ',
       skipLinkText: 'እቲ ትሕዝቶ ዝለል',
```
Comment (AlistairGempf) [inline src/app/lib/config/services/tigrinya.js]: This has changed in the spreadsheet.

Comment (DenisHdz) [reply src/app/lib/config/services/tigrinya.js]: mmm I see the same one `ህሉው ገጽ` in the spreadsheet.

Comment (AlistairGempf) [reply src/app/lib/config/services/tigrinya.js]: Sorry, this is for the skipLinkText: `ናብቲ ትሕዝቶ ቀጽል`

Comment (DenisHdz) [reply src/app/lib/config/services/tigrinya.js]: Done! They are faster than us :P

**Comment by AlistairGempf** [comment]
They are too quick :D
Urdu:
`Related Content` has been updated

Vietnamese:
404 message, 500 title, 500 message and `photoGallery` have all been updated
The international privacy message is also still in english but the UK one is in vietnamese - I'm guessing they should be the same.

Tigrinya:
International cookie has no space after the hyperlink (`international/cookie/last`), which it might need.
Call to actions in error messages are in `[]`s which I think probably need to be removed.

**Comment by DenisHdz** [comment]
> They are too quick :D
> Urdu:
> `Related Content` has been updated
> 
> Vietnamese:
> 404 message, 500 title, 500 message and `photoGallery` have all been updated
> The international privacy message is also still in english but the UK one is in vietnamese - I'm guessing they should be the same.
> 
> Tigrinya:
> International cookie has no space after the hyperlink (`international/cookie/last`), which it might need.
> Call to actions in error messages are in `[]`s which I think probably need to be removed.

Done. I have also updated the `Call to actions` translations.

**Comment by AlistairGempf** [comment]
2 small things in Tigrinya:
- `photoGallery` translation has changed
- Trust URL has changed

**Comment by DenisHdz** [comment]
> 2 small things in Tigrinya:
> 
>     * `photoGallery` translation has changed
> 
>     * Trust URL has changed

Updated.

**Review by AlistairGempf — APPROVED** [review]
All looks good to me now!

**Comment by OlgaLyubin** [comment]
Seems like a link for Urdu for "Why you can trust BBC" in the footer should be replaced with https://www.bbc.com/urdu/institutional-49282935

File: src/app/lib/config/services/vietnamese.js [hunk 3]
```diff
@@ -40,130 +40,133 @@ const service = {
...
     },
     footer: {
       externalLink: {
         href: 'https://www.bbc.co.uk/help/web/links/',
-        text: 'Read about our approach to external linking.',
+        text: 'Tìm hiểu cách chúng tôi tiếp cận việc dẫn tới trang ngoài',
       },
       links: [
         {
           href: 'https://www.bbc.com/news/help-41670342',
```
Comment (OlgaLyubin) [inline src/app/lib/config/services/vietnamese.js]: There is a new link for this: https://www.bbc.com/vietnamese/institutional-49283563

Comment (DenisHdz) [reply src/app/lib/config/services/vietnamese.js]: Done

**Review by OlgaLyubin — APPROVED** [review]
Look good to me

**Review by FK78 — APPROVED** [review]

**Review by thekp — APPROVED** [review]

**Review by dr3 — APPROVED** [review]
LGTM, needs testing

**Comment by paruchurisilpa** [comment]
Looks good to me...

**Pull request merged**